### PR TITLE
Updated pom.xml

### LINKED ISSUE #4: DPU cannot be imported
When I try to import the DPU, the following error is shown:

```
Failed to load DPU because of exception:org.osgi.framework.BundleException: The bundle "eu.comsode.plugins.uv-e-skGovCRP_1.0.0.SNAPSHOT [53]" could not be resolved. Reason: Missing Constraint: Import-Package: net.sf.ehcache; version="0.0.0"
```

Context: UV from https://github.com/UnifiedViews/Core/commit/1d30de3b63fac33afeec9dcb2f8859feee6d9a06.

### PR DESCRIPTION
**Pull request by mbl-eea-sk**
In order to correct dependencies.

Resolves #4

### CODE REVIEW

**Comment by mbl-eea-sk** [comment]
Test please.

File: pom.xml [hunk 1]
```diff
@@ -36,13 +36,12 @@
 		<dependency>
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
-			<scope>compile</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpclient-osgi</artifactId>
 			<version>4.3.3</version>
```
Comment (skrchnavy) [inline pom.xml:43]: when DPU is provided it shall be defined in section https://github.com/UnifiedViews/Plugin-DevEnv/blob/UV_Plugin-DevEnv_v2.1.3/uv-pom-dpu/pom.xml#L71
Or when any library is expe3cted to be provided out of scope standard UV installation, it must be documented in Readm.md / Technical notes.

File: pom.xml [hunk 2]
```diff
@@ -77,4 +75,41 @@
 		</repository>
 	</repositories>
 
+	<build>
```
Comment (skrchnavy) [inline pom.xml]: Not clear why is added this part into pom.xml

Comment (mbl-eea-sk) [inline pom.xml]: Eclipse claims an error with lifecycle configuration. In fix offers it generated this part. As the DPU works without it as well I actually removed it.

**Comment by jindrichmynarz** [comment]
Using fd8e7c047219249d99c1d8136e5b4801527d53e1 the error remains the same.

**Comment by jindrichmynarz** [comment]
I forgot to checkout the feature branch. Retesting...

**Comment by jindrichmynarz** [comment]
256e5edb999f0055b42b3ef1432c6e91f211d7fa imports fine. The pull request can be merged.

File: pom.xml (hunk 1 above)
Comment (mbl-eea-sk) [inline pom.xml:43]: It was a revert to original state. Seems I accidentally declared it as compiled in https://github.com/comsode-uv-plugins/e-skGovCRP/commit/fd8e7c047219249d99c1d8136e5b4801527d53e1.

**Pull request merged**